# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/clang/mips64.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- clang-14.0.1.src/lib/Basic/Targets/Mips.h.vanilla	2022-06-16 14:44:15.725439802 +0200
+++ clang-14.0.1.src/lib/Basic/Targets/Mips.h	2022-06-16 14:48:00.861426383 +0200
@@ -75,7 +75,7 @@
     else
       setABI("n64");
 
-    CPU = ABI == "o32" ? "mips32r2" : "mips64r2";
+    CPU = ABI == "o32" ? "mips32r2" : "mips4";
 
     CanUseBSDABICalls = Triple.isOSFreeBSD() ||
                         Triple.isOSOpenBSD();
